modules/WebcastPolls/resources/webcastPollsKalturaProxy.js

```diff
@@ -126,84 +126,142 @@
 
             return defer.promise();
         },
-        transmitVoteUpdate : function(metadataId, userId, selectedAnswer,pollId)
+        transmitVoteUpdate : function(metadataId, userId, selectedAnswer,pollId ,newApi)
```

Comment:
@kalturaguy  - is there a point of leaving the old functionality? As I see it - BE supports both, versions > V2.58 will work with new API, <=2.57 with old API 
I did this new flag so our QA will be able to test the both DVR and new API without conflicts. 
Do you see a reason to keep legacy too?